presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix poor query planning performance for queries that are submitted when the cluster does not have the minimum required number of workers available.
+* Fix an issue causing a union of a table with a ``VALUES`` statement to execute on a single machine, which may result in out of memory errors.
+* Improve the logical plan by removing unnecessary remote exchanges before window functions when the data is already available locally.
+* Add grouped execution support for window functions.
+* Add support for the ``BOOLEAN`` type for ``EXPLAIN IO`` statements.
+* Add support for choosing distribution type for semi joins based on the cost model when ``join_distribution_type='AUTOMATIC'``.
+* Add support for collecting table statistics on demand with the ``ANALYZE`` statement.
+* Add a config option (``query.stage-count-warning-threshold``) to specify a per-query threshold for the number of stages.
+  When this threshold is exceeded a ``TOO_MANY_STAGES`` warning is raised as part of the query information.
+* Add per-task peak user memory usage to query statistics.
+* Add the number of bytes spilled to the presto-cli output.
+* Add :func:`ST_Points` function.
+* Add :doc:`/connector/elasticsearch`.
+* Remove the system memory pool and related configuration properties (``resources.reserved-system-memory``, ``deprecated.legacy-system-pool-enabled``) entirely.
+  System memory pool was deprecated in 0.201, and it was unused by default since that release. All memory allocations will now be served from the general/user memory pool.
+
+
+Web UI
+------
+
+* Improve live plan to show data transfer statistics as edge labels.
+* Improve live plan to contain more information about each node. For example, table scan nodes now show the name of the tables that are being scanned.
+* Improve stage skew histograms to make it easier to identify low skew.
+* Add the ability to zoom when viewing plans for completed queries.
+* Add the number of bytes spilled to query details page.
+
+
+Hive Connector Changes
+----------------------
+
+* Fix an issue that may cause a partially successful rollback to cause data loss and corrupt the metastore.
+  The ``hive.skip-target-cleanup-on-rollback`` config option can be used to skip deleting target directories when partition creation is rolled back.
+* Fix an issue where creating a table on S3 could fail for empty S3 "directories"
```

Comment:
I think I understand why you used quotes here, but I'm not sure anybody else well. I would lose the quotes or explain further.

Reply:
@raghavsethi I don't think I have the context. Since this was @kokosing 's commit, I went through his commit and added the release notes for him.

Reply:
How about:

```
Fix an issue where creating a table on S3 could fail for S3 prefixes without any associated objects (e.g., empty S3 directories).
```